pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -354,7 +365,10 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                     formatted='{pokemon} capture failed.. trying again!',
                     data={'pokemon': pokemon.name}
                 )
-                sleep(2)
+
+                # sleep according to flee_count and flee_duration config settings
+                sleep(choice([(x+1)*self.config.catchsim_flee_duration for x in range(self.config.catchsim_flee_count)]))
```

Comment:
Yeah something like that but handling floats.  I think I can replace that line with something more obvious and probably more efficient like:

`randrange(flee_count) * flee_duration`

Hopefully that's more obvious.  I can add a line comment too to explain what's going on here but I think that's a much better line =]